presto-main/src/main/java/io/prestosql/server/HttpRequestSessionContext.java

```diff
@@ -642,12 +635,10 @@ private static WebApplicationException badRequest(String message)
     }
 
     /**
-     * Check and return a given string if it is non-null and non-empty.
+     * Check, trim and return a given string if it is non-null and non-empty.
      *
      * @param value a given string value.
-     *
-     * @return {@code string} the given string if it is non-null and non-empty
-     *
+     * @return the given string trimmed if it is non-null and non-empty
```

Comment:
```suggestion
     * @return the given string trimmed if it is neither null nor empty
```
What happens if the given string is either null or empty, though?

Reply:
done